example/android/app/build.gradle

```diff
@@ -168,11 +168,12 @@ repositories {
 dependencies {
     // The version of react-native is set by the React Native Gradle Plugin
     implementation("com.facebook.react:react-android")
+    implementation 'com.google.android.material:material:1.4.+'
```

Comment:
is this necessary?

Reply:
yes
![error](https://github.com/dojo-engineering/react-native-pay-sdk/assets/91316159/0a75e7bb-1eb1-4c11-b057-70e4574be424)